docs/linux-requirements.md

```diff
@@ -0,0 +1,62 @@
+
+# Linux
+
+## Requirements to build native modules
+
+If you want to build native modules, make sure that the following tools are installed on your system.
+
+- [Node 14](https://nodejs.org)
+
+You can run the following commands to install Node 14:
+```
+curl -sL https://deb.nodesource.com/setup_14.x | sudo bash -
+sudo apt -y install nodejs
+```
+
+Verify the version of Node.js installed. (Latest version is preferred)
+```
+node -v
+```
+
+Install the Node dev tools and the yarn package manager using the following commands:
+```
+sudo apt -y install gcc g++ make
+curl -sL https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
+echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
+sudo apt update && sudo apt install yarn
+```
+
+Verify the version of Yarn installed. (Must be v1)
+```
+yarn -v
+```
+
+- [Python 3](https://www.python.org/downloads/)
+
+You can run the following commands to install Python 3:
+```
+sudo apt-get install python3.6
+python3 --version
+```
+
+- [Strawberry Perl](https://strawberryperl.com/)
```

Comment:
Should there be instructions here?

Reply:
> Should there be instructions here?

I made this file corresponding to `Windows requirements`: https://github.com/vector-im/element-desktop/blob/develop/docs/windows-requirements.md and I personally did not need to install `Strawberry Perl` on Linux system to build the native modules, but still have provided the link to its installation documentation looking at the prerequisites for Windows.

Reply:
You should remove "Strawberry Perl" and replace it with Perl, with instructions like:

```bash
sudo apt-get install perl
```